obi_one/scientific/simulation/stimulus.py

```diff
@@ -349,3 +356,85 @@ def _generate_config(self) -> dict:
 
 class SynchronousSingleSpikeStimulus(Stimulus):
     spike_probability: float | list[float]
+
+
+class SpikeStimulus(Stimulus):
+    _module: str = "synapse_replay"
+    _input_type: str = "spikes"
+    duration: float
```

Comment:
Should accept lists as well:
```suggestion
    duration: float | list[float]
```